src/test/java/edu/hm/hafner/analysis/parser/GhsMultiParserTest.java

```diff
@@ -105,10 +105,16 @@ void issue59118() {
                 .hasCategory("#1729-D")
                 .hasSeverity(Severity.WARNING_NORMAL);
     }
+    @Test
```

Comment:
Hey @uhafner 

I have added this test case. 

It catches one line as expected. Could you confirm if the bug is exposed?

Also, what would be the ideal solution here? Do I have to fix the [GHS_MULTI_WARNING_PATTERN  ](https://github.com/Jagrutiti/analysis-model/blob/1ac5fd9eaaf4125db7c52d14a3fd81471b82bda1/src/main/java/edu/hm/hafner/analysis/parser/GhsMultiParser.java#L26)?

Reply:
Yes the test is correctly failing. You can improve the test by checking the expected properties as well. But this can be done in the second step. 

You need to check why the regular expression that already contains a column match does not work. Either the pattern is broken or the code that reads the pattern.